eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -198,6 +207,21 @@ noSuchGovActions gas gaIds =
    in failureUnless (Set.null unknownGovActionIds) $
         GovActionsDoNotExist unknownGovActionIds
 
+checkVotesAreValid ::
+  ConwayEraPParams era =>
+  Map.Map (GovActionId (EraCrypto era)) (GovActionState era) ->
+  VotingProcedures era ->
+  Test (ConwayGovPredFailure era)
+checkVotesAreValid gas (VotingProcedures vps) =
+  let go voter gmap =
+        let foldMapFailures = flip foldMap' . Map.toList . fmap gasAction . Map.restrictKeys gas $ Map.keysSet gmap
+            voterMismatchUnless check (govActionId, govAction) = failureUnless (check govAction) $ VoterMismatch voter govActionId
```

Comment:
This line is also way longer than our common line, so maybe you can split it on two lines

Reply:
Thanks!